packages/env/src/index.ts

```diff
@@ -8,7 +9,7 @@ import z from 'zod'
 const environment = process.env.NODE_ENV || 'development'
 
 const __dirname = fileURLToPath(import.meta.url)
-const pathToEnv = resolve(__dirname, `../../.env.${environment}`)
+const pathToEnv = resolve(cwd(), `../../.env.${environment}`)
```

Comment:
Was this wrong?

Reply:
yeah

Reply:
__dirname varies between apps so it's not consistent, cwd is consistent